WordPress/gutenberg
# Comment Content: Add Border Support

## What?

Add border block support to the `Comment Content` block.

Part of https://github.com/WordPress/gutenberg/issues/43247

## Why?
`Comment Content` block is missing border support.

## How?

Adds the border block support in block.json.

## Testing Instructions

- Go to Global Styles Settings ( Under Appearance > Editor > Styles > Edit styles > Blocks ).
- Make sure that `Comment Content` block's border is Configurable via Global Styles.
- Edit template/page, Add  `Comment Content` block and Apply the border Styles.
- Verify that `Comment Content` block styles take precedence over global Styles.
- Verify that `Comment Content` block borders display correctly in both the Editor and Frontend.

## Screenshots or Screencast 
 

https://github.com/user-attachments/assets/20194239-0eaf-45ba-bfb6-342d3d0585d8
